src/components/Hero/Hero.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './Hero.css';
 
-const Hero = ({ title, image, displayMovieDetail, id }) => {
+const Hero = ({ title, image }) => {
   return (
-    <div className="hero" onClick={() => { displayMovieDetail(id); }} tabIndex="0">
+    <div className="hero" tabIndex="0">
```

Comment:
looks way cleaner without the onClick event. We should see if we still need tabIndex. Hero is now wrapped in a link so may automatically be tab-able

Reply:
Oh good point! We may not.